### PR TITLE
Fix bug with filename computation.

### PR DESCRIPTION
**Pull request by sookocheff**
Fix a bug introduced during some cleanup.

### CODE REVIEW

File: autoload/test/java/maven.vim [hunk 1]
```diff
@@ -15,8 +18,6 @@ function! test#java#maven#build_position(type, position) abort
       return [filename]
```
Comment (sookocheff) [inline autoload/test/java/maven.vim]: Bug here: filename may not be defined if we can't find a nearest match.

**Comment by janko-m** [comment]
Looks good. Could you just squash it into a single commit?

**Comment by sookocheff** [comment]
@janko-m Squashed. Thank you.

**Comment by janko-m** [comment]
Great, thanks again :)

**Pull request merged**